tests/test_includes.py

```diff
@@ -153,3 +157,11 @@ def test_monorepo_runs_each_task_with_expected_cwd(
     else:
         assert result.stdout.endswith("/tests/fixtures/monorepo_project/subproject_2\n")
     assert result.stderr == ""
+
+    result = run_poe_subproc("get_cwd_3", project="monorepo", cwd=projects["example"])
```

Comment:
You can pass `--root` to make this test pass like so:
```suggestion
    result = run_poe_subproc(
        "--root", 
        str(projects["monorepo/subproject_3"]), 
        "get_cwd_3", 
        cwd=projects["example"]
    )
```